apps/client/src/pages/builder/sidebars/left/dialogs/profiles.tsx

```diff
@@ -83,14 +108,14 @@ export const ProfilesDialog = () => {
               <FormControl>
                 <div className="flex items-center gap-x-2">
                   <Avatar className="size-8 bg-white">
-                    {field.value && (
+                    {iconSrc && (
                       <AvatarImage
                         className="p-1.5"
-                        src={`https://cdn.simpleicons.org/${field.value}`}
+                        src={iconSrc}
                       />
                     )}
                   </Avatar>
-                  <Input {...field} id="iconSlug" placeholder="linkedin" />
+                  <Input {...field} id="iconSlug" placeholder="linkedin" onChange={(e) => handleIconChange(e, field)} />
```

Comment:
Personally, if the only reason to pass `field` to the `handleIconChange` function is to call `field.onChange(event)`, you could just create an anonymous function inline to call it separately. So it would look something like this:

```suggestion
                  <Input {...field} id="iconSlug" placeholder="linkedin" onChange={(event) => {
                    field.onChange(event);
                    handleIconChange(event);
                  }} />
```

This way, you don't need to pass the field (and it's complex types) to the function.